test/makeup_json_test.exs

```diff
@@ -2,101 +2,215 @@ defmodule MakeupJsonTest do
+      #
+      # assert length(tokens) == 6
+      # assert {:name_tag, _, ^string} = Enum.at(tokens, 1)
+    end
+  end
+
+  test "Escape sequences in JSON string values are parsed correctly" do
+    strings = [
+      "\"",
+      "\\",
+      "/",
+      "b",
+      "f",
+      "n",
+      "r",
+      "t",
+      "u0123",
+      "u4567",
+      "u89ab",
+      "ucdef",
+      "uABCD",
+      "uEF01"
+    ]
+
+    for string <- strings do
+      assert [{:string_double, _, ["\""]}, {:string_escape, _, _}, {:string_double, _, ["\""]}] =
+               JsonLexer.lex(~s("\\#{string}"))
+
+      # TODO: What this should be, according to the pygments tests:
+      # https://github.com/pygments/pygments/blob/master/tests/test_data.py#L145
+      #
+      # assert [{:string_double, _, ^string}] = JsonLexer.lex(~s("\\#{string}"))
+    end
+  end
+
+  test "Single-line comments are tokenized correctly" do
+    text = ~s({"a//b"//C1\n:123/////C2\n}\n// // C3)
+    tokens = JsonLexer.lex(text)
+
+    assert {:comment_single, _, ["//", "C", "1"]} = Enum.at(tokens, 2)
+    assert {:comment_single, _, ["//", "/", "/", "/", "C", "2"]} = Enum.at(tokens, 6)
+    assert {:comment_single, _, ["//", " ", "/", "/", " ", "C", "3"]} = Enum.at(tokens, 10)
+
+    assert tokens |> Enum.filter(fn {type, _, _} -> type == :comment_single end) |> length() == 3
+
+    # Input and output texts must match!
+    assert tokens |> Enum.map(fn {_, _, content} -> to_string(content) end) |> Enum.join() == text
+  end
+
+  test "Multi-line comments are tokenized correctly" do
+    text = ~s(/** / **/{"a /**/ b"/* \n */:123})
+    tokens = JsonLexer.lex(text)
+
+    assert {:comment_multiline, _, "/** / *"} = Enum.at(tokens, 0)
+    assert {:comment_multiline, _, "*/"} = Enum.at(tokens, 1)
+    assert {:comment_multiline, _, "/* "} = Enum.at(tokens, 4)
+    assert {:comment_multiline, _, " "} = Enum.at(tokens, 6)
+    assert {:comment_multiline, _, "*/"} = Enum.at(tokens, 7)
+
+    assert tokens |> Enum.filter(fn {type, _, _} -> type == :comment_multiline end) |> length() ==
+             5
+
+    # TODO: What those probably should be(?):
+    # https://github.com/pygments/pygments/blob/master/tests/test_data.py#L179
+    #
+    # assert {:comment_multiline, _, "/** / **/"} = Enum.at(tokens, 0)
+    # assert {:comment_multiline, _, "/* \n */"} = Enum.at(tokens, 3)
+    # assert tokens |> Enum.filter(fn {type, _, _} -> type == :comment_multiline end) |> length() == 2
+
+    # Input and output texts must match!
+    assert tokens |> Enum.map(fn {_, _, content} -> to_string(content) end) |> Enum.join() == text
+  end
+
+  test "Unfinished or unclosed single-line comments are parsed as errors" do
+    assert [{:error, _, ?/}] = JsonLexer.lex("/")
+    assert [_number, {:error, _, ?/}] = JsonLexer.lex("1/")
+    assert [{:error, _, ?/}, _number] = JsonLexer.lex("/1")
+    assert [_string, {:error, _, ?/}] = JsonLexer.lex("\"\"/")
+  end
+
+  # TODO:
+  @tag :skip
```

Comment:
for multi line comment were incorrectly handled. I have a pending pr. Once this is merged, I think we can uncomment this.

Reply:
PR is merged. :skip tag can be removed.